Core/MIPS/MIPS.h

```diff
@@ -210,8 +211,20 @@ class MIPSState
 		struct {
 			u32 pc;   //241
 
-			u32 lo;   //242
-			u32 hi;   //243
+			union {
+#ifdef COMMON_BIG_ENDIAN
+				struct {
+					u32 hi;   //242
+					u32 lo;   //243
```

Comment:
Hmm.  I wonder if this will cause an unexpected surprise elsewhere... at least need to make ThreadContext in sceKernelThread.h match.

-[Unknown]

Reply:
oh that's a good catch ! I could update that as well but then there are the alignment + savestate problems caused by reading/writing to the others[6] array directly, hmm..
I wonder if I should just keep the old order and just assign lo/hi directly with masking/shifting.

Reply:
ok I have decided to go with the safer but less optimal solution.
messing with the context layout has just way too many ramifications, and I can't figure out all of them right now heh.

Reply:
You're right - that's better.  Almost forgot about save states.  I guess that'd be one way to test the audio, heh.

-[Unknown]